### PR TITLE
Generate feed depending of source of content

### PR DESCRIPTION
**Pull request by smakosh**
## Summary
- Generate feed (RSS file) depending on source of content

## Fixes
https://github.com/narative/gatsby-theme-novela/issues/123

### CODE REVIEW

File: @narative/gatsby-theme-novela/gatsby-config.js [hunk 1]
```diff
@@ -51,22 +52,55 @@ module.exports = ({
...
+                      ...edge.node,
+                      description: edge.node.excerpt,
+                      date: edge.node.date,
+                      url: site.siteMetadata.siteUrl + edge.node.slug,
+                      guid: site.siteMetadata.siteUrl + edge.node.slug,
+                      // custom_elements: [{ "content:encoded": edge.node.body }],
+                      author: edge.node.author,
+                    };
+                  });
+              } else {
```
Comment (mabasic) [inline @narative/gatsby-theme-novela/gatsby-config.js]: Can you explain what does this code do? I get that first you check local then contentful, but this like I am unsure. You mix both?

File: @narative/gatsby-theme-novela/gatsby-config.js [hunk 2]
```diff
@@ -81,7 +115,58 @@ module.exports = ({
...
+                      author {
+                        name
+                      }
+                      secret
+                    }
+                  }
+                }
+              }
+              `
+                : `
```
Comment (mabasic) [inline @narative/gatsby-theme-novela/gatsby-config.js]: Also here. What is this for? I don't understand that is why I am asking.

**Comment by mabasic** [comment]
Btw. Very nice solution how you managed to get two queries inside the plugin.

File: @narative/gatsby-theme-novela/gatsby-config.js (hunk 1 above)
Comment (smakosh) [reply @narative/gatsby-theme-novela/gatsby-config.js]: > With the flexibility of Gatsby, Novela is able to pull different data sources to build your website. Right now we support Contentful and your local file system. All data sources can be combined together or used seperately.

https://github.com/narative/gatsby-theme-novela#data-sources

File: @narative/gatsby-theme-novela/gatsby-config.js (hunk 2 above)
Comment (smakosh) [reply @narative/gatsby-theme-novela/gatsby-config.js]: Explained above ☝️

Comment (mabasic) [reply @narative/gatsby-theme-novela/gatsby-config.js]: Got it. Seems fine to me. Haven't tested, but it looks good.

Comment (smakosh) [reply @narative/gatsby-theme-novela/gatsby-config.js]: Thank you! I appreciate your feedback and co-operation

**Pull request merged**